### PR TITLE
usm: nodejs: Fix linter issues

### PR DESCRIPTION
**Pull request by guyarb**
### What does this PR do?

Addresses two linter issues found in #31216

### Motivation

Cleanup TODOs in the code

### CODE REVIEW

File: pkg/network/usm/nodejs.go [hunk 1]
```diff
@@ -163,7 +166,7 @@ func (m *nodeJSMonitor) Stop() {
 }
 
 // getNodeJSPath checks if the given PID is a NodeJS process and returns the path to the binary
```
Comment (amitslavin) [inline pkg/network/usm/nodejs.go:168]: The documentation does not fit the function name isNodeJSBinary

**Review by amitslavin — APPROVED** [review]

File: pkg/network/usm/nodejs.go (hunk 1 above)
Comment (guyarb) [reply pkg/network/usm/nodejs.go]: thanks, fixed

**Pull request merged**